### DISCUSSION TITLE
Is there a Json file available with the GTM4WP GA4 events/parameter/triggers/tags available?

### DISCUSSION THREAD

**Discussion by VictorNorbart** [duracelltomi/gtm4wp#221] [General]
Hi @duracelltomi Thomas, 

We have been using your plugin with much pleasure, satisfaction and success for many years. 

Now, we were wondering if there was a Json file available, comprised of the GTM container with all the GA4 e-commerce tracking tags, triggers, parameters etc. 

Normally, I install all the ecommerce tracking things manually. However, when I work with Magento, Shopify and Shopware, Github has offered me almost instant integration possibilities by providing GTM container Json files. 

I am new to github, so perhaps I missed the file. 

Do you have such a file available?

Thanks a lot in advance, 

Kind regards and greeting from the Netherlands.

Victor (big fan of your work)

**Comment by VictorNorbart** [comment]
I found the solution here: https://gtm4wp.com/how-to-articles/how-to-setup-enhanced-ecommerce-tracking-google-analytics-4-ga4-version You can download the container via that link

**Reply by snoofs** [reply]
gtm4wp website is down. Do you have the json file to share?

**Reply by duracelltomi** [reply]
I've got a report from Brazil that they can not access the website. I checked my GA stats and indeed, there are some countries where I do not have any visitors. I am looking into this, until then you can use a VPN to get az IP rage from the US or any EU country.
